types/redux-form/lib/selectors.d.ts

```diff
@@ -2,6 +2,7 @@ import { FormErrors, GetFormState } from "../index";
 
 export type DataSelector<FormData = {}, State = {}> = (formName: string, getFormState?: GetFormState) => (state: State) => FormData;
 export type ErrorSelector<FormData = {}, State = {}, ErrorType = string> = (formName: string, getFormState?: GetFormState) => (state: State) => FormErrors<FormData, ErrorType>;
+export type GetFormErrorSelector<State = {}, ErrorType = string> = (formName: string, getFormState?: GetFormState) => (state: State) => ErrorType;
```

Comment:
The one place this is used, the `getFormError` export, passes no type parameters, so the type is always essentially
```ts
export type GetFormErrorSelector = (formName: string, getFormState?: GetFormState) => (state: {}) => string;
```
since the generics seem extraneous, maybe it's best to just write it like that?